<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_islem</class>
 <widget class="QMainWindow" name="gestion_islem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/new/prefix1/img/maxresdefault.jpg);
background-image: url(:/new/prefix1/img/maxresdefault.jpg);
background-color: rgb(230, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>711</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/img/maxresdefault.jpg);</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Commande</string>
     </attribute>
     <widget class="QLineEdit" name="rech_commande">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Recherche</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableView" name="table_commande">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>71</y>
        <width>651</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#d54e21;&quot;&gt;Gestion Commande&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="commande_sel_col">
      <property name="geometry">
       <rect>
        <x>591</x>
        <y>30</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>All</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Client</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Attente</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Date</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>410</y>
        <width>281</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Imprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>410</y>
        <width>331</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Export Excel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Vente</string>
     </attribute>
     <widget class="QTableView" name="table_vente">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>651</width>
        <height>341</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#d54e21;&quot;&gt;Gestion Vente&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rech_vente">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Recherche</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="vente_sel_col">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>30</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>All</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Commande</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Prix</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Livraison</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Payement</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>420</y>
        <width>281</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Imprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>420</y>
        <width>331</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Export Excel</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>737</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/img/maxresdefault.jpg);</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionadd_commande"/>
   <addaction name="actiondelete_commande"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionadd_vente"/>
   <addaction name="actiondelete_vente"/>
   <addaction name="separator"/>
   <addaction name="actiondeconnecter"/>
  </widget>
  <action name="actionadd_commande">
   <property name="icon">
    <iconset>
     <normaloff>img/ajouter article.jpg</normaloff>
     <normalon>:/new/prefix1/img/ajouter article.jpg</normalon>img/ajouter article.jpg</iconset>
   </property>
   <property name="text">
    <string>add_commande</string>
   </property>
  </action>
  <action name="actiondelete_commande">
   <property name="icon">
    <iconset>
     <normaloff>img/supprimer article.jpg</normaloff>
     <normalon>:/new/prefix1/img/supprimer article.jpg</normalon>img/supprimer article.jpg</iconset>
   </property>
   <property name="text">
    <string>delete_commande</string>
   </property>
  </action>
  <action name="actionadd_vente">
   <property name="icon">
    <iconset>
     <normaloff>img/Add-icon.png</normaloff>
     <normalon>:/new/prefix1/img/Add-icon.png</normalon>img/Add-icon.png</iconset>
   </property>
   <property name="text">
    <string>add_vente</string>
   </property>
  </action>
  <action name="actiondelete_vente">
   <property name="icon">
    <iconset>
     <normaloff>img/delete-icon.png</normaloff>
     <normalon>:/new/prefix1/img/delete-icon.png</normalon>img/delete-icon.png</iconset>
   </property>
   <property name="text">
    <string>delete_vente</string>
   </property>
  </action>
  <action name="actiondeconnecter">
   <property name="icon">
    <iconset>
     <normaloff>img/Windows-Log-Off-icon.png</normaloff>
     <normalon>:/new/prefix1/img/Windows-Log-Off-icon.png</normalon>img/Windows-Log-Off-icon.png</iconset>
   </property>
   <property name="text">
    <string>Se deconnecter</string>
   </property>
   <property name="toolTip">
    <string>deconnecter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
